pylint/reporters/multi_reporter.py

```diff
@@ -95,8 +95,14 @@ def on_set_current_module(self, module: str, filepath: Optional[AnyPath]) -> Non
 
     def on_close(
         self,
-        stats: Mapping[Any, Any],
-        previous_stats: Mapping[Any, Any],
+        stats: Dict[
```

Comment:
I changed this to `Dict` instead of `Mapping` as its only ever called with `self.stats` and using `Mapping` gave a `Mypy` warning because of an incorrect argument type for one of the calls to `on_close`.